.changeset/tender-penguins-admire.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
@augustjk I suppose we should include the starter kits here, since updating dev dependencies is more relevant than for our own packages.

I so wonder if we should mention the rollup and terser updates for our packages as well though, since conceivably they could effect people if something goes wrong. wdyt?